src/accountsdb/db.zig

```diff
@@ -72,7 +72,7 @@ pub const AccountsDB = struct {
     unrooted_accounts: RwMux(SlotPubkeyAccounts),
 
     // pubkey->account LRU maps
-    accounts_cache: RwMux(AccountsCache),
+    maybe_accounts_cache_mux: ?RwMux(AccountsCache),
```

Comment:
the convention weve been following elsewhere is `_rw` for `RwMux`

Reply:
few places where _mux is still used, eg:  `const accounts_cache_mux = &(self.maybe_accounts_cache_rw orelse break :blk null);`

Reply:
https://github.com/Syndica/sig/pull/325/commits/cf31bb0ad2fed66f340c14b84582ab47fcb50f3f